src/index.js

```diff
@@ -256,7 +271,18 @@ function validateSchema (config) {
       region: {type: 'string'},
       secrets: secretsArraySchema
     },
-    required: ['region', 'secrets'],
+    required: ['secrets'],
+    additionalProperties: false
+  };
+
+  const defaultsObjectSchema = {
+    type: 'object',
+    properties: {
+      secretNamePrefix: {type: 'string'},
```

Comment:
what are use-cases for these defaults? Especially for `ignoreSecretNamePrefix`?
This is open source project. Use-case should be common for all